auth/authentication.js

```diff
@@ -1,19 +1,17 @@
 const validateToken = require('./validate-token');
 const config = require('../config');
-exports.register = function (server, options, next) {
-  server.state('token', {
-    isSecure: false
-  });
-  server.auth.strategy('jwt', 'jwt', false,
-    {
-      key: config.JWT_SECRET,
-      validateFunc: validateToken,
-      verifyOptions: { ignoreExpiration: true }
+exports.plugin = {
+  name: 'Authentication',
+  register: async function (server, options) {
+    server.state('token', {
+      isSecure: false
```

Comment:
consider making this an environment variable with a default to `false` so that it can be configured in AWS without a code release. (_it's OK for now_)